### DISCUSSION TITLE
which scale is used to measure arousal before delirium assessment？

### DISCUSSION THREAD

**Discussion by scorpioneuro** [MIT-LCP/mimic-code#1766] [MIMIC-IV]
Hello Support Team,
CAM-ICU is used for delirium assessment in MIMIC-IV.
Prior to delirium assessment, the patient's level of arousal should be assessed to determine if the patient is appropriate for delirium assessment. We found that there are two sedation scales, RASS and Rick-SAS, that are used to measure arousal in the MIMIC database.  
I would like to know which scale is used to measure arousal before delirium assessment.  
As far as we know, RASS is the most commonly  tool used to assess arousal prior to delirium assessment.  Is RASS also used in the MIMIC database?

Thank you!

**Comment by aircondition426** [comment]
I also have the same issue with you. Each item under the RASS scale has information collected at multiple time points, and there is no detailed scoring record in it. There is only Yes or No label. Does Yes mean positive for the event and No mean negative for the event? There are multiple time points in one item of the same icu-stay, including Yes or No. Is there an event that is judged as positive in only one time point? The incidence of delirium in that way is very high, even to more than 90%, which does not seem to be consistent with the highest incidence of delirium mentioned in the literature I read as high as 20%.
